chapters/002-smol-internet/README.md

```diff
@@ -105,6 +138,25 @@ In the output from `ip route`, we can see `default via 10.1.2.1 dev eth0`, which
 identifies that as the default gateway. We're seeing these requests in our
 tcpdump because the ARP cache needs to periodically be refreshed.
 
+> **Wait... what's the difference between `ip addr` and `ip route`?**
+
+There's some similar output between the `ip addr` and `ip route` commands. `ip addr` gives us view into the network interfaces available on a host.
```

Comment:
~available on a host~ 
perhaps: "available on a **machine**", if we distinguish a "host" as "something that does not route"